docs/pages/distribution/uploading-apps.md

```diff
@@ -15,49 +13,58 @@ To learn how to build native binaries, see [Building Standalone Apps](../buildin
-    7. Make a note of the filename of the JSON file downloaded to your computer. You'll need this to upload your app later. Be sure to keep this JSON file secure, as it provides API access to your Google Play developer account.
-5. Return to the **API access** page on the **Google Play Console** and ensure it shows your new service account.
-6. Click on **Grant Access** for the newly added service account
-7. Choose **Release Manager** from the newly added service account
-8. Click **ADD USER** to close the dialog
+See [expo.fyi/creating-google-service-account](https://expo.fyi/creating-google-service-account) to learn more.
 
 #### Manually uploading your app for the first time
 
-Before using `expo-cli` for uploading your standalone app builds, you have to upload your app manually at least once. [See here for the instructions on how to do it.](https://support.google.com/googleplay/android-developer/answer/113469)
+Before using `expo upload:android` for uploading your standalone app builds, you have to upload your app manually at least once.<br>
+See [expo.fyi/first-android-submission](https://expo.fyi/first-android-submission) to learn more.
 
 #### Using expo-cli to upload the further builds of your app
 
 After these steps, you can make use of `expo-cli` to upload your further app builds to Google Play.
 
-To upload your Android app to Google Play, run `expo upload:android`. You can set following options when uploading an Android standalone app:
-- `--key <key>` **(required)** - path to the JSON key used to authenticate with the Google Play Store (created in the previous steps)
+To upload your Android app to Google Play, simply run `expo upload:android`. You can set following options when uploading an Android standalone app:
+
+- `--type <archive-type>` - the archive type, by default, it's inferred from the filename extension, choose from: apk, aab
+- `--key <key>` - path to the JSON key used to authenticate with the Google Play Store
 - `--track <track>` - the track of the application to use, choose from: production, beta, alpha, internal, rollout (default: internal)
+- `--release-status <release-status>` - release status (used when uploading new apks/aabs), choose from: completed, draft, halted, inProgress (default: completed)
+- `--android-package <android-package>` - the Android package of your app, if you don't pass this parameter it'll be read from `app.json`
+- `--use-submission-service` - **Experimental:** Use Submission Service for uploading your app. The upload process will happen on Expo servers.
+- `--verbose` - always print logs from Submission Service
+
+#### Uploading your app with Submission Service
+
+**Beware:** This feature is still experimental!
+
+If you'd like to upload your app from any other operating system than macOS, you can give Submission Service a try. Submission Service is a part of Expo Application Services. It lets you upload your standalone app directly from Expo Servers.
```

Comment:
```suggestion
If you would like to upload your app from any other operating system than macOS, you can give the Expo Submission Service a try. It lets you upload your standalone app directly from Expo Servers.
```

Reply:
i only removed Expo Application Services here because we haven't used the term anywhere else on our website yet and so it could confuse people if we start to use it before we have some other marketing material or docs for it